app/controllers/alchemy/admin/contents_controller.rb

```diff
@@ -32,9 +32,10 @@ def update
       end
 
       def order
-        params[:content_ids].each do |id|
-          content = Content.find(id)
-          content.move_to_bottom
+        Content.transaction do
+          params[:content_ids].each_with_index do |id, idx|
+            Content.where(id: id).update_all(position: idx + 1)
```

Comment:
If this only produces one SQL query, then I'm fine with it. Otherwise we need to live with that.